src/Extension/FluentExtension.php

```diff
@@ -1055,6 +1055,14 @@ public function LocaleInformation($locale = null)
             $localeObj = Locale::getDefault();
         }
 
+        if (!$locale && !$localeObj) {
```

Comment:
```suggestion
        if (!$localeObj) {
```
We only need the `$localeObj` don't we?

Reply:
Possibly, though the issue I was fixing was specifically for code path where `$locale = null` so it. 

I've moved the code so that it's underneath Locale::getDefault() just to show the intent better. This seems like the most place I can put this that will have the least side-effects. I don't know if there will be any unintended negative side-effects also creating a fake locale for when `Locale::getByLocale($locale)` returns null, or creating a fake locale inside Locale::getDefault().

Reply:
Updated